### ISSUE TITLE
Warning when launching docker-compose

### ISSUE THREAD

**Issue by Ala-Na** [Reaster0/ft_transcendence#139]
Just to keep you up to date

Front
-----
- Some warning about npm packages use in front (will try to download latest versions from my computer as I don't have sudo rights on 42 computers)
`npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@achrinza/node-ipc@9.2.2',
npm WARN EBADENGINE   required: { node: '8 || 10 || 12 || 14 || 16 || 17' },
npm WARN EBADENGINE   current: { node: 'v18.2.0', npm: '8.10.0' }
npm WARN EBADENGINE }
npm WARN deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated vuetify-loader@2.0.0-alpha.9: vuetify-loader has been renamed to webpack-plugin-vuetify for Vuetify 3
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.`

- Some warning about size of files used

`warning  

asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  fonts/materialdesignicons-webfont.5d875350.eot (1000 KiB)
  fonts/materialdesignicons-webfont.54b0f60d.woff2 (318 KiB)
  fonts/materialdesignicons-webfont.d671cbf6.woff (454 KiB)
  fonts/materialdesignicons-webfont.21f691f0.ttf (1000 KiB)
  img/arrow-left.50d7b8ba.png (375 KiB)
  img/arrow-right.28c52cca.png (374 KiB)
  img/blurredy.d059c8a3.jpg (3.3 MiB)
  css/chunk-vendors.05ccbf40.css (591 KiB)
  js/chunk-vendors.b7d63461.js (439 KiB)

 warning  

entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  app (1.11 MiB)
      css/chunk-vendors.05ccbf40.css
      js/chunk-vendors.b7d63461.js
      css/app.d3e08c8a.css
      js/app.dae8f6ed.js

  File                                   Size              Gzipped

  dist/js/chunk-vendors.b7d63461.js      439.12 KiB        133.54 KiB
  dist/js/app.dae8f6ed.js                89.08 KiB         21.30 KiB
  dist/js/webfontloader.33ec0995.js      12.03 KiB         4.84 KiB
  dist/css/chunk-vendors.05ccbf40.css    591.02 KiB        84.45 KiB
  dist/css/app.d3e08c8a.css              17.17 KiB         2.94 KiB
`

Back
----
- npm packages warning (same as front, will try to upload latest versions)

`npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated crypto@1.0.1: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.
npm WARN deprecated @types/socket.io@3.0.2: This is a stub types definition. socket.io provides its own type definitions, so you do not need this installed.
npm WARN deprecated @hapi/pinpoint@2.0.0: Moved to 'npm install @sideway/pinpoint'
npm WARN deprecated @hapi/formula@2.0.0: Moved to 'npm install @sideway/formula'
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @hapi/address@4.1.0: Moved to 'npm install @sideway/address'
npm WARN deprecated @hapi/joi@17.1.1: Switch to 'npm install joi'`

- Vulnerabilities : Will look into it, but that's probably the recent nestJS vulnerabilities (we can do nothing about it)
`12 high severity vulnerabilities`

**Comment by Reaster0** [comment]
yeah i know about theses warning, but actually you have them when you install the dependencies by yourself the first time, it's just the life of the js develloper, having deprecated dependencies

**Comment by Ala-Na** [comment]
#140 PR clean some of the warnings. I actually managed to obtain a version with no npm warnings but for back it means to use typeorm 0.3 (we're under 0.2-something) and many request to db doesn't use the same syntax. Changing syntax is tricky, so it's better to keep it as it is now. And I don't remember exactly how I cleaned all warning on front ^^'
